tests/all/stack_creator.rs

```diff
@@ -0,0 +1,157 @@
+#[cfg(all(not(target_os = "windows"), not(miri)))]
+mod not_for_windows {
+    use anyhow::{bail, Context};
+    use std::{
+        alloc::{GlobalAlloc, Layout, System},
+        ops::Range,
+        ptr::NonNull,
+        sync::Arc,
+    };
+    use wasmtime::*;
```

Comment:
Can remove a level of indentation via `#![...]` to put the attribute on the whole `stack_creator` module, rather than defining a submodule inside it:

```suggestion
#![cfg(all(not(target_os = "windows"), not(miri)))]

use anyhow::{bail, Context};
use std::{
    alloc::{GlobalAlloc, Layout, System},
    ops::Range,
    ptr::NonNull,
    sync::Arc,
};
use wasmtime::*;
```